test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -748,24 +751,431 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			}
 		})
 
-		AfterEach(func() {
-			By("Reverting the Profile")
-			spec, err := json.Marshal(initialProfile.Spec)
+		It("[test_id:50964] Offline Higher CPUID's", func() {
```

Comment:
what's the purpose of this test?
is aligned with the selection in ppc? https://github.com/openshift/cluster-node-tuning-operator/pull/354

Reply:
The purpose of this test is to check whether we can offline Higher cpu ids . Specifically on numa machines.